lib/executors/Simulator.types.ts

```diff
@@ -10,6 +10,8 @@ export type TransactionParameters = {
   max_priority_fee_per_gas?: number;
   value: string | number;
   access_list?: AccessList;
+  save?: boolean;
```

Comment:
I would avoid extending `TransactionParameters` type with simulation specific fields. Maybe we could Create a separate type for `SimulationTransactionParameters` or extend/change `SimulationCallArguments` that would accept these two params. 🤔